core/server/worker/src/main/java/alluxio/worker/grpc/BlockReadHandler.java

```diff
@@ -87,10 +87,8 @@
   private static final Logger SLOW_BUFFER_LOG = new SamplingLogger(LOG, Constants.MINUTE_MS);
   private static final long SLOW_BUFFER_MS =
       ServerConfiguration.getMs(PropertyKey.WORKER_REMOTE_IO_SLOW_THRESHOLD);
-  /** Metrics. */
   private static final Counter RPC_READ_COUNT =
-      MetricsSystem.counterWithTags(MetricKey.WORKER_ACTIVE_RPC_READ_COUNT.getName(),
-            MetricKey.WORKER_ACTIVE_RPC_READ_COUNT.isClusterAggregated());
+      MetricsSystem.counter(MetricKey.WORKER_ACTIVE_RPC_READ_COUNT.getName());
```

Comment:
isn't `WORKER_ACTIVE_RPC_READ_COUNT.isClusterAggregated() == true`, and thus this metrics will report to cluster? will your patch change the behavior?

Reply:
My patch didn't change the behavior, this metrics will be reported to master. Just that this metric don't need to use counterWithTags since there is no tags.

Reply:
@LuQQiu From what I remember, the reason we used `counterWithTags` was to register the counter with the `SHOULD_REPORT_METRICS` in the `MetricsSystem`
- https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/metrics/MetricsSystem.java#L497-L503

Without this, the metric didn't appear in the master's log sink. @yuzhu has context on this as well.

Reply:
@LuQQiu From what I remember, the reason we used `counterWithTags` was to register the counter with the `SHOULD_REPORT_METRICS` in the `MetricsSystem`
- https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/metrics/MetricsSystem.java#L497-L503

Without this, the metric didn't appear in the master's log sink. @yuzhu has context on this as well.